docs/md2ipynb.py

```diff
@@ -4,7 +4,8 @@
 import notedown
 import nbformat
 
-assert len(sys.argv) == 2, 'usage: input.md'
+print(sys.argv)
+assert len(sys.argv) == 3, 'usage: input.md True'
```

Comment:
Seems that the CI server cannot print
This code works well on my local machine, but it fails on the server. Could you help look into what's the difference for the environment on the server, resulting in the difference of `sys.argv`? @leezu